pkg/cmd/server/api/v1/types.go

```diff
@@ -1149,3 +1152,10 @@ type AdmissionConfig struct {
 	// on the master. Order is significant. If empty, a default list of plugins is used.
 	PluginOrderOverride []string `json:"pluginOrderOverride,omitempty"`
 }
+
+// ControllerConfig holds configuration values for controllers
+type ControllerConfig struct {
```

Comment:
> Maybe this should be specific to service signing. We have other blocks that configure other individual controllers.

This was an attempt to start fixing that historical mistake, but I suppose I'd rather have a `ControllerConfig` that contained a `ServiceServingCertSignerConfig` if that would satisfy.